src/wasm-binary.h

```diff
@@ -1251,11 +1252,7 @@ class WasmBinaryWriter {
       : data(data), size(size), pointerLocation(pointerLocation) {}
   };
 
-  std::vector<Buffer> buffersToWrite;
-
   void emitBuffer(const char* data, size_t size);
```

Comment:
Can we delete `emitBuffer` and `buffersToWrite`, too? If not, why can we delete `finishUp`?

Reply:
buffersToWrite is deleted right?   (I forgot to remove the declaration of `emitBuffer`.. will do that now).